userguide/one_shot_object_detection/README.md

```diff
@@ -109,16 +101,14 @@ starter_images = tc.SFrame({'image':[tc.Image('stop_sign.png')],
 my_backgrounds = tc.SArray('my_custom_backgrounds.sarray')
 
 # Create a model using custom background images                                    
-model = tc.one_shot_object_detector.create(starter_images, 'label', backgrounds = my_backgrounds)
+model = tc.one_shot_object_detector.create(starter_images, 'label', backgrounds=my_backgrounds)
 ```
 
-To view the synthetic images generated with your custom background images:
+To preview the synthetic images generated with your custom background images:
```

Comment:
Repeated synthetic images section. I'd show it only once.

Reply:
The difference here is the added third argument which is `my_backgrounds` when calling `preview_synthetic_training_data`. I thought it might be helpful to show the change in code needed to preview synthetic images when using custom backgrounds. I was thinking it's better to be thorough over brevity.